runtime/src/accounts_db.rs

```diff
@@ -354,13 +361,11 @@ pub struct AccountsDB {
     /// Keeps tracks of index into AppendVec on a per slot basis
     pub accounts_index: RwLock<AccountsIndex<AccountInfo>>,
 
-    /// Account storage
     pub storage: RwLock<AccountStorage>,
 
     /// distribute the accounts across storage lists
     pub next_id: AtomicUsize,
 
-    /// write version
```

Comment:
A bit aggressive here, but remove this comment as it adds no additional information to the commented code itself. use fewer comments and remove otherwise as it's too common for engineers to maintain them up-to-date (including me!) as [this](https://github.com/solana-labs/solana/pull/7559/files#r359687698) is the case.